recirq/qml_lfe/learn_dynamics_C.py

```diff
@@ -0,0 +1,167 @@
+import os
+import cirq
+import cirq_google
+import numpy as np
+import circuit_blocks
+
+from absl import app
+from absl import flags
+from absl import logging
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer("n", None, "System size (total qubits == 2 * n).")
+flags.DEFINE_integer("depth", None, "Circuit depth (block-wise).")
+flags.DEFINE_integer(
+    "n_data",
+    20,
+    "Number of circuits generated for each class (total circuits == 2 * n_data).",
+)
+
+flags.DEFINE_integer(
+    "batch_size",
+    5,
+    "Number of circuits to send over the wire per batch (value does not affect results).",
+)
+
+flags.DEFINE_integer(
+    "n_shots", 2000, "Number of measurements to draw from each individual circuit."
+)
+
+flags.DEFINE_string(
+    "save_dir", "./data", "Path to save experiment data (must already exist)."
+)
+
+flags.DEFINE_bool("use_engine", False, "Whether or not to use quantum engine.")
+
+
+def flatten_circuit(circuit):
```

Comment:
Ideally add typing information if you get a chance.

Reply:
Added.